index.html

```diff
@@ -4,164 +4,188 @@
-          <li>Comunity</li>
+    <header class="container">
+      <div class="navbar">
+        <a href="#" class="logo-href"><img src="./assets/image/logo.jpg" alt="" class="img-logo"></a>
+        <ul class="nav-ul">
+            <li class="nav-item"><a class="nav-link" href="#">Home</a></li>
+            <li class="nav-item"><a class="nav-link" href="#">About</a></li>
+            <li class="nav-item"><a class="nav-link" href="#">Service</a></li>
+            <li class="nav-item"><a class="nav-link" href="#">Poduct</a></li>
+            <li class="nav-item"><a class="nav-link" href="#">Events</a></li>
+            <li class="nav-item mr-4"><a class="nav-link" href="#">community</a></li>
         </ul>
-      </nav>
+        <div class="hamburger">
+          <i class="fas fa-bars hamburger-icon text-center"></i>
+        </div>
+      </div>
     </header>
-    <main>
-      <div class="">
-        <div id="demo" class="carousel slide" data-ride="carousel">
-          <!-- Indicators -->
-          <ul class="carousel-indicators">
-            <li data-target="#demo" data-slide-to="0" class="active"></li>
-            <li data-target="#demo" data-slide-to="1"></li>
-            <li data-target="#demo" data-slide-to="2"></li>
-          </ul>
-          <!-- The slideshow -->
-          <div class="carousel-inner">
-            <div class="carousel-item active">
-              <img src="la.jpg" alt="Los Angeles">
-            </div>
-            <div class="carousel-item">
-              <img src="chicago.jpg" alt="Chicago">
-            </div>
-            <div class="carousel-item">
-              <img src="ny.jpg" alt="New York">
-            </div>
-          </div>
-          <!-- Left and right controls -->
-          <a class="carousel-control-prev" href="#demo" data-slide="prev">
-            <span class="carousel-control-prev-icon"></span>
-          </a>
-          <a class="carousel-control-next" href="#demo" data-slide="next">
-            <span class="carousel-control-next-icon"></span>
-          </a>
+    <main class="container">
+      <div class="contain">
+        <div class="container">
+          <img class="mySlides" src="./assets/image/bg1.jpg" style="width:100%" alt="">
+          <img class="mySlides" src="./assets/image/bg2.jpg" style="width:100%" alt="">
+          <img class="mySlides" src="./assets/image/bg5.jpg" style="width:100%" alt="">
         </div>
+        <button class="carousel-btn-1" onclick="plusDivs(-1)">&#10094;</button>
+        <button class="carousel-btn-2" onclick="plusDivs(1)">&#10095;</button>
       </div>
-      <div class="">
+      <div class="container mt-5">
         <div class="">
           <div class="">
-
+            <h6 class="text-center h1 text-lightblue">Offers</h6>
           </div>
         </div>
-        <div class="">
-          <div class="">
-
-          </div>
-          <div class="">
-
-          </div>
-          <div class="">
-
+        <div class="container">
```

Comment:
repeating container several times, its possible to wrap within one.remember DRY?

Reply:
container is a class name. and i called it because we need it to control the flow of our project. And the dry concept--- as far as we know, We are not repeating any css attribute here. If we are wrong please  provide us some trusted documents.